CHANGELOG.rst

```diff
@@ -1,6 +1,7 @@
 **0.1.2 - TBD**
```

Comment:
What version do we want this to be? I believe this is the last ticket for this epic.

Reply:
I think this is correct